cypress/support/config/settings.js

```diff
@@ -179,12 +179,12 @@ module.exports = () => ({
             enabled: false,
           },
           test: {
-            paths: [],
-            enabled: false,
+            paths: ['/afaanoromoo/sport-23125110'],
+            enabled: true,
```

Comment:
I would probably not enable all test FIX pages - this will add to the runtime of our tests in CI. 

Let's see if we can come up with a list of services which would be good examples e.g. RTL, LTR, variants etc. 

We could probably use the same services as Front Page? 

I can help you with this.